src/NHSD.BuyingCatalogue.Organisations.Api/Controllers/OrganisationsController.cs

```diff
@@ -157,7 +157,7 @@ public async Task<ActionResult<CreateOrganisationResponseModel>> CreateOrganisat
 
         [HttpGet]
         [Route("{id}/service-recipients")]
-        public async Task<ActionResult> GetServiceRecipientsAsync(Guid id)
+        public async Task<ActionResult<IEnumerable<ServiceRecipientsModel>>> GetServiceRecipientsAsync(Guid id)
```

Comment:
I forgot to mention, now that you've made this change you can simplify the return statement to just `return model;`.

Reply:
👍